frontend/isoko/src/components/imageUpload/cropImage.js

```diff
@@ -0,0 +1,95 @@
+export const createImage = (url) =>
+   new Promise((resolve, reject) => {
+      const image = new Image();
```

Comment:
I'm not gonna review this file assuming it's just the file you copied from the package right?